websites/K/Komik Indo/presence.ts

```diff
@@ -0,0 +1,129 @@
+const presence = new Presence({
+		clientId: "1272521587912081428",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/HccQ60t.png",
+	Home = "https://i.imgur.com/m6Cw03l.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Browsing...",
+			largeImageKey: Assets.Logo,
+			smallImageKey: Assets.Logo,
```

Comment:
The `smallImageKey` might not be needed in this case, since it is just the logo.